portfolio/src/main/webapp/blogScript.js

```diff
@@ -1,27 +1,52 @@
 window.addEventListener('DOMContentLoaded', (event) => {
+    getURL();
+    getPost();
     getComments();
 });
 
-function getComments() {
-    fetch('/data').then(response => response.json()).then((comments) => {  
-      const commentListElement = document.getElementById('comment-list');
-      for (comment of comments) {
-        commentListElement.appendChild(createCommentElement(comment));
+const tagIndx = parent.document.URL.substring(parent.document.URL.indexOf('tag='), parent.document.URL.length);
```

Comment:
`parent` here means `window.parent` (https://developer.mozilla.org/en-US/docs/Web/API/Window/parent)

But you're not in an iframe or anything like that. You can just use document.